themes/bootstrap3/js/check_item_statuses.js

```diff
@@ -203,9 +204,11 @@ VuFind.register('itemStatuses', function ItemStatuses() {
     for (var i = 0; i < ajaxItems.length; i++) {
       checkItemStatus($(ajaxItems[i]));
     }
+
+    VuFind.emit("item-status-done");
```

Comment:
Is it really right to emit this status here? Isn't checkItemStatus queuing up work? If so, don't we want to emit this when the work is actually done, not when we're done creating the queue?